src/Utilities/copy.jl

```diff
@@ -608,6 +627,8 @@ function try_constrain_variables_on_creation(
     single_variable_not_added
 end
 
+function final_touch(::MOI.ModelLike, idxmap) end
```

Comment:
This needs a docstring.

Reply:
Let me know what you think for the name. Another option would be `end_of_copy`